README.md

```diff
@@ -1 +1,38 @@
-# portfolio-mobile-first
\ No newline at end of file
+# https://github.com/eyoab01/portfolio-mobile-first
+This page is about my portfolio as a software developer
+
+## Built With
+HTML and CSS
+
+
+## Setup
+download linters folder from microverse github and copy it to your project.
+
+## Install
+install the linters and the fontawesome library. (Preferred)
+
+To get a local copy up and running follow these simple steps.
+
+## Prerequisites
+Install code editor to be able to run the commands in the terminal. VSCode 
+## Setup
+ download the linters folder and copy it to your project You can clone the project through git clone https://github.com/eyoab01/portfolio-mobile-first
+
+## Install
+Install the linters folder to your project. Install the fontawesome for icons in the project 
+
+## Deployment link
+
+My portifolio website can be viewed at:  [https://github.com/eyoab01/portfolio-mobile-first]
+
+
+### Authors
+👤 Tesfaye Teklu Feyissa
+
+GitHub: @eyoab01
+Twitter: @tesfayeteklu10
+LinkedIn: www.linkedin.com/in/tesfaye-teklu-352067105
+
+Feel free to check the issues page.
+
+
```

Comment:
- [ ] kindly update the README file of your project - please use [this template](https://github.com/microverseinc/readme-template) as required in the project but customize it to your project and add also the license it will give a small description and documentation to your project.  :+1: